app/index.js

```diff
@@ -25,7 +25,7 @@ var versions = {
   'master': 'master',
   '0.4.0': 'v0.4.0',
   '0.4.1': 'v0.4.1',
-  '0.4.2': 'v0.4.2'
+  '0.4.2': '0.4.2'
```

Comment:
Just on my twitter/fb page of the project, nothing too serious.
I'm ok with deleting it.